agent/util-scripts/pbench-register-tool-set

```diff
@@ -101,6 +101,9 @@ case "$toolset" in
 			nerrs=$nerrs+1
 		fi
 	done
+        ansible-galaxy install chaitanyaenr.ara
```

Comment:
As the requestor of ara being added to pbench, I will add the context about how this tool would be used and why:

- We need a way to profile ansible runs.  ara is not feature complete for these needs, but it's the best we've found to do this.
- ara itself has an install phase which ends in ansible being setup to monitor ansible runs (in our case an openshift installation).
- The register-tool bit is similar to the --install option that other benchmark scripts have.  I don't think it necessarily needs to live in register-tool.

I do agree that the implementation of this new tool should match all existing tool patterns.